cmd/snap/cmd_snap_op.go

```diff
@@ -270,6 +271,7 @@ func (mx modeMixin) asksForMode() bool {
 type cmdInstall struct {
 	channelMixin
 	modeMixin
+	Revision string `long:"revision" description:"Install the given revision (only for own snaps)"`
```

Comment:
`"Install the given revision of personal snap"`?

Is the store complaining nicely when that's not the case?